package.json

```diff
@@ -35,7 +35,7 @@
     "catw": "^1.0.1",
     "connect-redis": "^3.0.1",
     "debug": "^2.2.0",
-    "elasticsearch": "^10.1.3",
+    "elasticsearch": "^12.1.3",
```

Comment:
Was unable to get search working with ES 5.x and older versions of the JS library.